lib/libc/math/lib_logl.c

```diff
@@ -62,30 +70,56 @@ long double logl(long double x)
 	long double y_old;
 	long double ey;
 	long double epsilon;
+	int iter, relax_factor;
```

Comment:
Please declare variable in separate lines as per coding guidelines.

Reply:
Modified